### ISSUE TITLE
Server Crash on Start-up with Sponge

### ISSUE THREAD

**Issue by jelaw21** [mcenderdragon/Futurepack-API#98]
I am using Futurepack 24.2.45 and its crashing the server before starting. I know its Futurepack because I pulled it from the pack and it loaded fine. Thoughts? Ideas?

[crash-2016-12-14_11.30.04-server.txt](https://github.com/mcenderdragon/Futurepack-API/files/652321/crash-2016-12-14_11.30.04-server.txt)

**Comment by mcenderdragon** [comment]
We already reported the Bug to sponge without any real response.
https://github.com/SpongePowered/SpongeForge/issues/1046

**Comment by jelaw21** [comment]
I commented at the above Bug too. REally need this to work!!!

**Comment by jelaw21** [comment]
Have you found ANY versions of Sponge that works?

**Comment by mcenderdragon** [comment]
Nope, they are doing simething wrong with Dimensions and we theoraticaly need sponge too (for our servers) But it seems like they also have a hard time fixing these Bugs.

**Comment by mcenderdragon** [comment]
Duplicate of https://github.com/mcenderdragon/Futurepack-API/issues/77
